app/ui.js

```diff
@@ -9,7 +9,7 @@
 import * as Log from '../core/util/logging.js';
 import _, { l10n } from './localization.js';
 import { isTouchDevice, isMac, isIOS, isAndroid, isChromeOS, isSafari,
-         hasScrollbarGutter, dragThreshold }
+         hasScrollbarGutter, dragThreshold, isAsyncClipboardAvailable }
```

Comment:
I wonder if we might need to expose this from the `RFB` object. Our app is not the only user, and this here is an internal function that others should not use.